src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -123,6 +124,10 @@ public function load(array $configs, ContainerBuilder $container)
                 ->replaceArgument(2, $sonataRoles);
         }
 
+        if (!isset($bundles['TwigExtraBundle'])) {
```

Comment:
Instead of checking the bundle is registered, I think we could check something more specific.
By instance: https://github.com/sonata-project/SonataIntlBundle/pull/302/files#diff-0501290a6a6012a0743dd122a7d2ff14R122-R137.
I prefer that kind of check since the service can be overridden by users and the bundle availability doesn't guarantee the service integrity.
WDYT?

Reply:
Sure, let me change it.

Reply:
So, should we require `twig/extra-bundle` or just suggest it?

Reply:
Based on https://github.com/sonata-project/SonataMediaBundle/pull/1756#discussion_r443121733, I'd say we should suggest this dependency.